client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -95,55 +97,137 @@ const ArtistHeaderSection: React.FC<{ artist: Artist; isManage?: boolean }> = ({
+                margin-bottom: 0rem;
               }
             `}
           >
             {artist.avatar?.sizes?.[300] && (
-              <img
-                src={artist.avatar?.sizes?.[120]}
-                alt="Artist avatar"
+              <div
                 className={css`
-                  margin-right: 1rem;
+                  max-width: 85px;
+                  display: flex;
+
+                  @media screen and (max-width: ${bp.medium}px) {
+                    max-width: 50px;
+                    padding-bottom: 0rem;
+                    margin-bottom: 0rem;
+                  }
                 `}
-              />
+              >
+                <img
+                  src={artist.avatar?.sizes?.[300]}
+                  alt="Artist avatar"
+                  className={css`
+                    width: 100%;
+                    border-radius: 100px;
+                    border: solid 1px var(--mi-lighter-foreground-color);
+                  `}
+                />{" "}
+              </div>
             )}
+
             <div
               className={css`
                 width: 100%;
+                display: flex;
+                ${artistAvatar ? "min-height: 85px;" : ""}
+                ${artistAvatar ? "margin-left: 1rem;" : ""}
+                flex-direction: column;
+                justify-content: center;
+                @media screen and (max-width: ${bp.medium}px) {
+                  ${artistAvatar ? "min-height: 50px;" : ""}
+                  ${artistAvatar ? "margin-left: .5rem;" : ""}
+                }
               `}
             >
               <HeaderDiv
                 className={css`
-                  align-items: flex-start !important;
+                  align-items: center !important;
+                  padding-bottom: 0 !important;
+                  @media screen and (max-width: ${bp.medium}px) {
+                    margin: 0rem !important;
+                  }
                 `}
               >
-                <div>
-                  <H1>{artist.name}</H1>
-                </div>
-                <div>
-                  {!isManage && <FollowArtist artistId={artist.id} />}
-                  {!isManage && user?.id === artist.userId && (
-                    <Link to={`/manage/artists/${artist.id}`}>
-                      <Button
-                        compact
-                        transparent
-                        type="button"
-                        startIcon={<FaPen />}
-                      >
-                        {t("edit")}
-                      </Button>
-                    </Link>
-                  )}
+                <div
+                  className={css`
+                    ${artistAvatar ? "min-height: 50px;" : ""}
+                    display: flex;
+                    align-items: center;
+                    justify-content: space-between;
+                    width: 100%;
+                  `}
+                >
+                  <div>
```

Comment:
This shouldn't need a <div> wrapper? If it needs to be set to `display: block` I would set it on the H1 component